.gitlab-ci.yml

```diff
@@ -251,7 +251,7 @@ trigger_pipeline:
 # Build jobs
 # Job with example runs.
 build/cuda90/gcc/all/debug/shared:
-  <<: *default_build_with_test
+  <<: *default_build
```

Comment:
I guess you plan to change this back to `default_build_with_test`  ?

Reply:
For now we cannot, because we do not have a GPU to run CUDA/HIP.

Reply:
Don't the bwuni/FH2 controllers have V100's ? Or are those only for Level 3 CI and you are using Level 1 right now ?

Reply:
Unfortunately no, those are just rebranded login nodes

Reply:
We are using Level 2 now, i.e. directly running on the controllers (login nodes). Level 3, we cannot access ourselves and need the admins to register the runner themselves.